third_party/clang-format/git-clang-format

```diff
@@ -171,6 +171,7 @@ def main():
       print('clang-format did not modify any files')
   elif opts.diff:
     print_diff(old_tree, new_tree)
+    sys.exit(1)
```

Comment:
Just query: You want to return failure only when we run this command with `--diff` option, right?

This looks reasonable but I'm not sure how upstream people think this change.
Can we leave a `FIXME(yjaeseok)` comment to revisit here later?

Reply:
@romandev Yes, this only for --diff option.

I will leave a ```FIXME``` comment